examples/simple/Error_Codes.csv

```diff
@@ -1,156 +1 @@
CV-TDG18,Declaration for appears with invalid storage class specifiers.,6.7.1:3,Constraint Violation
CV-CDE1,Enum initializer not an integer constant expression.,6.7.2.2:2,Constraint Violation
CV-CID1,"Trying to look up identifier , but no such identifier is in scope.",6.5.1:2,Constraint Violation
CV-TEAS1,Incompatible types in assignment or function call arguments.,6.5.16.1:1,Constraint Violation
CV-TEAS2,Incompatible pointer types in assignment or function call arguments.,6.5.16.1:1,Constraint Violation
CV-TEAS3,lvalue' required as left operand in assignment.,6.5.16:2,Constraint Violation
CV-TEAS5,Non-constant static initializer.,6.7.9:4,Constraint Violation
CV-TEAS6,Trying to write through a const lvalue.,6.5.16.1:1,Constraint Violation
CV-CEC1,"If both of a conditional expression's branches have pointer type, they must be compatible, or one must be a pointer to an object and the other must be a pointer to void.",6.5.15:3,Constraint Violation
CV-CEC5,"If one of a conditional expression's branches has pointer type and the other has integer type, the integer must be a null pointer constant.",6.5.15:3,Constraint Violation
CV-CEC2,"If one of a conditional expression's branches has void type, the other must also have void type.",6.5.15:3,Constraint Violation
CV-CEC4,"If one of a conditional expression's branches has struct or union type, the other must have the same type.",6.5.15:3,Constraint Violation
CV-EFC1,Function call with incomplete or invalid return type.,6.5.2.2:2,Constraint Violation
CV-TES1,Trying to compute sizeof of an expression with incomplete or a function type.,6.5.3.4:1,Constraint Violation
CV-TES2,Trying to compute sizeof of an incomplete or function type.,6.5.3.4:1,Constraint Violation
CV-FD1,Duplicate parameters in function definition.,6.7:3,Constraint Violation
CV-FD2,Invalid return type in function definition.,6.7.6.3:1,Constraint Violation
CV-FD3,Incomplete parameter type in function definition.,6.7.6.3:4,Constraint Violation
CV-SSA1,Non-constant expression in static assertion.,6.7.10:2,Constraint Violation
CV-SSA2,Static assertion failed: ,6.7.10:2,Constraint Violation
CV-CTI3,The width of a bit field shall be an integer constant expression.,6.7.2.1:4,Constraint Violation
CV-CTI2,Arrays must have positive length.,6.7.6.2:1,Constraint Violation
IMPL-CCV2,Conversion to signed integer outside the range that can be represented.,"6.3.1.3:3, J.3.5:1 item 4",Implementation Specific Behavior
IMPL-CEB8,The left operand in a bitwise right-shift is negative.,"6.5.7:5, J.3.5:1 item 5",Implementation Specific Behavior
IMPL-EIO5,Trying to modify an object with thread or auto storage duration in a thread other than the one in which the object is associated.,"6.2.4:4, 6.2.4:5, J.3.13:1 item 2",Implementation Specific Behavior
IMPL-EIO6,Attempting to access an object with thread or auto storage duration in a thread other than the one in which the object is associated.,"6.2.4:4, 6.2.4:5, J.3.13:1 item 2",Implementation Specific Behavior
```

Comment:
Anyway, we should not mention RV-Match in this file, because this is just the C semantics, which is public and can be used in other projects, too.  I would say instead:
```
The C11 semantic errors follow the template: Error_Type-Error_Code.
```